docs/source/model_design/concept_model/ensemble_distributions/index.rst

```diff
@@ -49,6 +49,119 @@ fits the data better than any of its individual components.
+plasma glucose, birthweight, or hemoglobin level. Suppose that we want to model
+:math:`E` for :math:`n` different population groups, e.g. indexed by (location,
+age, sex, year) as in GBD. Let :math:`j\in \{1,2,\ldots,n\}` specify which
+population group we are referring to.
+
+Typically, a GBD team will have microdata on :math:`E` for at least some of the
+population groups, and they are able to estimate the mean :math:`\mu_j` and
+standard deviation :math:`\sigma_j` of :math:`E` for each group :math:`j` using
+various regression methods. The goal is to use all the available microdata plus
+the estimates of :math:`\mu_j` and :math:`\sigma_j` to reconstruct the
+approxmiate distribution function of :math:`E` for each group :math:`j`.
+
+More precisely, the goal is to find a family of cumulative distribution
+functions :math:`F(x | \mu, \sigma)`, parameterized by mean :math:`\mu` and
+standard deviation :math:`\sigma`, satisfying the following criteria:
+
+* For each group :math:`j` that has microdata, the distribution function
+  :math:`F(x | \mu_j, \sigma_j)` provides a good approximation of the
+  `empirical CDF`_ of :math:`E` ; and
+
+* For all groups :math:`j`, the function :math:`F(x | \mu_j, \sigma_j)` provides
+  a plausible estimate of the population distribution of :math:`E` --- in particular, its mean and
+  standard deviation match :math:`\mu_j` and :math:`\sigma_j`.
+
+.. _empirical CDF: https://en.wikipedia.org/wiki/Empirical_distribution_function
+
+A simple strategy for estimating exposure distributions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+One simple approach to finding such a family of distributions :math:`F(x | \mu,
+\sigma)` would be to choose a 2-parameter family of probability distributions on
+:math:`\mathbb{R}`, such as the family of normal distributions
+:math:`\mathcal{N}(\mu,\sigma^2)`, and to fit distributions from that family to
+the exposure data using the `method of moments`_. That is, we would set
+:math:`F(x | \mu, \sigma) \equiv \mathcal{N}(\mu,\sigma^2)` so that :math:`E`
+would be normally distributed for each population group, with mean :math:`\mu_j`
+and standard deviation :math:`\sigma_j` for the population group :math:`j`.
+
+.. _method of moments: https://en.wikipedia.org/wiki/Method_of_moments_(statistics)
+
+Improving the distribution estimates with an ensemble approach
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. todo::
+  **Fill in this section.** Here are some unfinished snippets I started:
+
+  The family of functions :math:`F(x | \mu, \sigma)` will be defined as a
+  mixture of a fixed set of base distributions :math:`F_1(x | \mu, \sigma),
+  F_2(x | \mu, \sigma),\ldots, F_k(x | \mu, \sigma)`  parameterized by mean
+  :math:`\mu` and standard deviation :math:`\sigma`, using a fixed global set of
+  weights :math:`\{w_i\}_1^k` for all :math:`\mu` and :math:`\sigma`. GBD calls
+  :math:`F(x | \mu, \sigma)` an **ensemble distribution**.
+
```

Comment:
I wonder does GBD modeler always construct their ensemble distribution using 14 distributions (k=14)? How to determine the number of K and what are best fit using subset of certain distributions. I think the next step is to describe the methods of calculating distribution specific weight.

Reply:
I noticed that you introduced a simple approach, only limited to distributions of 2-paramater family, what about the mixture of 3-parameter and 2-parameter?

Reply:
It looks like only 11 of the 14 distributions are actually used in the current version of the ensemble code -- in particular, neither of the 3-parameter distributions are used (though it looks like the generalized lognormal was previously used).  I don't know how the ensemble fitting would deal with 3-parameter distributions instead of 2-parameter.

Some teams (e.g. the anemia team) may use a custom version of ensemble distributions that use fewer than the 14 (or 11) listed here. If the modelers use the ensemble fitting code from the Central Comp repo (see https://github.com/ihmeuw/vivarium_research/pull/479 for a description of what's in this repo), then weights will be calculated for all 11 distributions, but some weights may be 0. Including extra distributions in the weight optimization shouldn't hurt because the weight can just be 0 or close to 0 if the distribution doesn't need to be used.